about/pricing.html.markerb

```diff
@@ -154,6 +154,28 @@ regionSelect.addEventListener("change", function () {
 
 For stopped Machines we charge only for the root file system (rootfs) needed for each Machine. Each 1GB of rootfs for a Machine stopped for 30 days is $0.15. The amount of rootfs needed is defined by your OCI image generated on your app plus a few [containerd](https://containerd.io/+external) tweaks on the underlying file system.
 
+### Machine Reservation Blocks
+
+We have a block-based commit scheme. Reserve a block of compute time, either for `performance` Machines or `shared` Machines, for a 40% discount. Reservations apply to any number of Machines in that class, in a specific region.
```

Comment:
"scheme" has negative connotations (maybe esp in the US? I don't know)

```suggestion
You can purchase resource block-based commitments. You'll get a 40% discount when you reserve a block of compute time, either for `performance` Machines or `shared` Machines in a specific region.
```
or something like that? Maybe you'll need to add back in the bit about "no limit to the number of Machines"? If I understood that correctly?

Reply:
or do we want to say "purchase reservation" as below rather than "commitments"? In any case we should choose one of "reservation", "block", or "commitment" as the thing being purchased.

Reply:
Rejigged. Essentially got rid of the first sentence, since it felt like it was repetitive given content of second sentence.